presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/BasePlanTest.java

```diff
@@ -167,22 +166,11 @@ protected void assertMinimallyOptimizedPlan(@Language("SQL") String sql, PlanMat
                         queryRunner.getStatsCalculator(),
                         queryRunner.getCostCalculator(),
                         ImmutableSet.of(new RemoveRedundantIdentityProjections())),
-                translateExpressions());
+                queryRunner.translateExpressions());
```

Comment:
add comments: to avoid assert plan failure not printing out plan (for https://github.com/prestodb/presto/issues/12885).